grpc/proto/space.proto

```diff
@@ -232,15 +239,6 @@ service SpaceApi {
     };
   }
 
-  // Copies a set of files in a new or existing bucket that's shared with a group of public keys
-  // Auxiliary method that internally calls CreateBucket, ShareBucketViaPublicKey, AddFile.
-  rpc CopyAndShareFiles(CopyAndShareFilesRequest) returns (CopyAndShareFilesResponse) {
```

Comment:
Again we won't need this given how the sharing architecture worked out to be but let me know if its better we leave it for future uses (can't think of any).